cmd/image-builder/main.go

```diff
@@ -891,6 +846,50 @@ func main() {
 	fmt.Println("Job's done.")
 }
 
+func generateTags(o options) {
+	// Get the absolute path to the dockerfile directory.
+	dockerfilePath, err := getDockerfileDirPath(o)
+	if err != nil {
+		fmt.Printf("Failed to get dockerfile path: %s", err)
+		os.Exit(1)
```

Comment:
General remark: I would try to avoid exiting from the process inside the functions other than main. Unless this is kind of panic. 
When reading main function, you are usually not assuming that control is not returned from function.
Like os.Exit(0) below.
So my preference is to return error from function and on the main level decide if you exit with 0, non-zero or you continue execution.